.drone/drone.yml

```diff
@@ -0,0 +1,152 @@
+---
+kind: pipeline
+name: Lint
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: lint
+  image: golangci/golangci-lint:v1.37.1
+  commands:
+  - apt-get update -y && apt-get install -y libsystemd-dev
+  - make lint
+
+---
+kind: pipeline
+type: docker
+name: Test
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: test
+  image: rfratto/seego
+  commands:
+  - apt-get update &&  apt-get install -y rubygems rpm nsis  apt-transport-https ca-certificates curl gnupg lsb-release
+  - gem install --no-document fpm
+  - mkdir -p /usr/local/go/bin
+  - wget -q https://golang.org/dl/go1.16.5.linux-amd64.tar.gz
+  - tar -C /usr/local -xzf go1.16.5.linux-amd64.tar.gz
+  - rm go1.16.5.linux-amd64.tar.gz
+  - export PATH=$PATH:/usr/local/go/bin
+  - make DRONE=true BUILD_IN_CONTAINER=false test
+depends_on:
+  - Lint
+
+---
+kind: pipeline
+type: docker
+name: Dist
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+- name: distribute
+  image: rfratto/seego
+  commands:
+  - apt-get update &&  apt-get install -y rubygems rpm nsis
+  - gem install --no-document fpm
+  - mkdir -p /usr/local/go/bin
+  - wget -q https://golang.org/dl/go1.16.5.linux-amd64.tar.gz
+  - tar -C /usr/local -xzf go1.16.5.linux-amd64.tar.gz
+  - rm go1.16.5.linux-amd64.tar.gz
+  - export PATH=$PATH:/usr/local/go/bin
+  - make DRONE=true BUILD_IN_CONTAINER=false RELEASE_TAG=v0.0.0 dist
+depends_on:
+  - Test
+---
+kind: pipeline
+type: docker
+name: Containerize
+platform:
+  os: linux
+  arch: amd64
+trigger:
+  event:
+  - tag
```

Comment:
We still want containers per main-branch commit (Tempo team relies on them). I think it boils down to:

Main branch commit:
- Test 
- Containerize (tag being `main-sha`)

Tag:
- Test 
- Containerize (tag being `vX.Y.Z`)
- Dist 
- Publish